doc/classes/bool.xml

```diff
@@ -4,7 +4,45 @@
 		Boolean built-in type.
 	</brief_description>
 	<description>
-		Boolean built-in type.
+		Boolean is a built-in type. It can represent any data type that is either a true or false value. You can think of it as on or off (1 or 0) switch. It's often used as part of programming logic in condition statements like [code]if[/code] statements.
+
```

Comment:
There should not be empty lines in the `<description>` apart from within codeblocks. Each line is a separate paragraph, so this is an empty paragraph (`<p></p>`).